### PR TITLE
[Netmanager] Added alert to data export

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- This is specifically for when there is a network failure or server issue.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### Screenshots
Screenshot 2022-07-25 181351

### CODE REVIEW

File: netmanager/src/views/pages/Download/Download.js [hunk 1]
```diff
@@ -319,7 +328,7 @@ const Download = (props) => {
         dispatch(
           updateMainAlert({
             show: true,
-            message: err,
```
Comment (Codebmk) [inline netmanager/src/views/pages/Download/Download.js:322]: This is an object and yet message expects a string. It was throwing a blank screen and some errors. Fixed that.

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by 123MwanjeMike — APPROVED** [review]
image
LGTM

**Review by FaithDaka — APPROVED** [review]

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @Codebmk

**Pull request merged**